book/visual-effects.md

```diff
@@ -274,15 +284,17 @@ documentation for more on the Skia API.
 def parse_color(color):
     if color == "white":
         return skia.ColorWHITE
-    elif color == "lightblue":
-        return skia.ColorSetARGB(0xFF, 0xAD, 0xD8, 0xE6)
-    # ...
+    elif color == "red":
```

Comment:
Why?

Reply:
Ok, right. We _need_ to add `red` because we use that for the address bar cursor. I guess we don't need to remove `lightblue` but I'm not sure if we use it either (though—we do use it in an example, but that example _also_ uses `lightgreen`...)

Reply:
I'm leaning toward just implementing hex color parsing. That way exercises can use hex codes and we don't need any other keyword colors.

It should be really easy; colors in Skia are literally 32-bit integers so it should be something like:

```
if color.startswith("#") and len(color) == 7:
    r = int(color[1:3], 16)
    g = int(color[3:5], 16)
    b = int(color[5:7], 16)
    return skia.Color(r, g, b)
```

But I think that would be a separate PR, during or after the big reorg.

Reply:
Ok. for now can you put lightblue back?

Reply:
Done. Can we merge this now?